src/qml/components/ChatMessageInput.qml

```diff
@@ -207,8 +207,7 @@ Control {
     Component.onCompleted: {
         if (Platform.isDesktop) {
             messageField.forceActiveFocus();
-        }
-        else if (Platform.isIos) {
+        } else if (Platform.isIos) {
             app.keyboardEventFilter.install(messageField)
```

Comment:
Optional fix: it's good moment to remove the C++ filter class, we don't keep not used code

Reply:
We are using this code for ios